apps/api/src/app/workflows-v2/usecases/patch-workflow/patch-workflow.usecase.ts

```diff
@@ -31,20 +31,17 @@ export class PatchWorkflowUsecase {
     });
   }
 
-  private patchWorkflowFields(persistedWorkflow, command: PatchWorkflowCommand) {
-    const transientWorkflow = { ...persistedWorkflow };
-    if (command.active !== undefined) {
+  private patchWorkflowFields(persistedWorkflow, command: PatchWorkflowCommand): NotificationTemplateEntity {
+    const transientWorkflow: NotificationTemplateEntity = { ...persistedWorkflow };
+    if (command.active !== undefined && command.active !== null) {
```

Comment:
not relevant to this pr:
make sure we check for nulls as well, and type correction.

Reply:
need to revert null check change, because it is incorrect due to the new FE-BE communication design decision from Friday.